docs/release-notes/8.4.4-4.md

```diff
@@ -0,0 +1,79 @@
+
+Improves the [Data masking](..//data-masking-overview.md) performance by introducing an internal term cache. The new cache speeds up lookups for `gen_blocklist()` and `gen_dictionary()` functions by storing dictionary data in memory. However, if the dictionary table is modified directly (outside of the proper functions), the cache may become out of sync. To fix this, use the `new masking_dictionaries_flush()` function.
+
+Changes also affect row-based replication: dictionary changes on the source server are replicated, but the term cache on the replica doesn’t update immediately. To address this, a new system setting, `component_masking_functions.dictionaries_flush_interval_seconds()`, can be set to automatically refresh the cache at specified intervals, helping replicas stay in sync.
+
+Find more detailed information in the [Data masking overview](..//data-masking-overview.md) and in the [Data masking component functions](..//data-masking-function-list.md).
+
+### MySQL 8.4.4
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.4.4 and included in Percona Server for MySQL are the following:
+
+* Certain IO buffer serializations triggered an assertion in debug builds, causing the system to hang. (Bug #37139618)
+
+* Dropping the primary key and adding a new `AUTO_INCREMENT` column as the primary key in descending order using the `INPLACE` algorithm resulted in failure. (Bug #36658450)
+
+* Queries using a descending primary key with the `index_merge` optimization sometimes returned incorrect results, including missing rows. (Bug #106207, Bug #33767814)
+
+* When large transactions were being received and applied, and a request to stop the replication channel was made using STOP REPLICA, MySQL failed to stop the channel properly. As a result, it stopped processing channel commands. Additionally, the server shutdown process did not complete gracefully, requiring either the MySQL process to be killed or the host system to be restarted. (Bug #115966, Bug #37008345)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.4.4 release notes].
+
+## Improvements
+
+* [PS-9148](https://perconadev.atlassian.net/browse/PS-9148): Extends the [Data masking](..//data-masking-overview.md) with new additions from MySQL 8.3.0 [Enterprise Data Masking and De-Identification Component Variables](https://dev.mysql.com/doc/refman/8.4/en/data-masking-component-variables.html).
+
+## Bug fixes
+
+* [PS-9391](https://perconadev.atlassian.net/browse/PS-9391): The replication broke with the error `HA_ERR_KEY_NOT_FOUND` when the `slave_rows_search_algorithms` were set to `INDEX_SCAN,HASH_SCAN`.
+
+* [PS-9416](https://perconadev.atlassian.net/browse/PS-9416): The error messages from the Key Management Interoperability Protocol (KMIP) component were not descriptive.
+
+* [9509](https://perconadev.atlassian.net/browse/PS-9509): Percona Server stoped tracking the `global_connection_memory` when using `thread_handling='pool-of-threads'`.
```

Comment:
Percona Server stoped
should be
Percona Server stopped

Reply:
fixed